src/sim/account_sim.py

```diff
@@ -155,3 +155,81 @@ def record_position(self, position_df, action_df) -> bool:
         self.action_writer.record_position(position_df, action_df)
         self.action_writer.write_position()
         return True
+
+
+    # See https://www.mql5.com/en/docs/python_metatrader5/mt5accountinfo_py for more info
+    def get_account_info(self) -> dict:
+        return {
+            "login": 25115284,
```

Comment:
I don't think we actually need these methods to return anything. We should have mocks on the frontend like you already have added to handle checking the website without any API calls. These can return NotImplementedErrors. We shouldn't ever call the simulation from the frontend.

Reply:
What about during integration tests?

Reply:
I think it's redundant to have the data both mocked in the frontend as well as here. We should just pick and do one, I think.